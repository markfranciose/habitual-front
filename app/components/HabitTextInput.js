import React, {Component} from 'react';
import {TextInput} from 'react-native';

export default class HabitTextInput extends Component {
  constructor(props) {
    super(props);
    this.state = { habitname: 'What' };
  }

  render() {
    return (
      <TextInput
        style={{height: 40, width: 200, borderColor: 'gray', borderWidth: 1, color: 'white'}}
        onChangeText={(text) => this.setState({text})}
        value={this.state.text}
      />
    );
  }
}
