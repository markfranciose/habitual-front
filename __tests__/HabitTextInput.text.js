import React from 'react';
import 'react-native';
import HabitTextInput from '../app/components/HabitTextInput';
import renderer from 'react-test-renderer';


describe('HabitTextInput', () => {

  test('it is true', () => {
    expect(true).toBe(true);
  });

  test('is rendered correctly', () => {
    const tree = renderer.create(<HabitTextInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
