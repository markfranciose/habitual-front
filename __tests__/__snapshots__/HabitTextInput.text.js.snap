// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HabitTextInput is rendered correctly 1`] = `
<TextInput
  onChangeText={[Function]}
  style={
    Object {
      "borderColor": "gray",
      "borderWidth": 1,
      "color": "white",
      "height": 40,
      "width": 200,
    }
  }
  value={undefined}
/>
`;
